src/api/common.ts

```diff
@@ -96,6 +96,7 @@ export const promiseTimeout = (
         return res(a);
       })
       .catch((e) => {
+        clearTimeout(id);
```

Comment:
Without clearing the timeout, failed requests were waiting for the duration of the timeout (60 seconds) before returning.